### PR TITLE
added line on pw changes; minor grammar changes

### PR DESCRIPTION
**Pull request by Saraislet**
I updated all gender references to be gender neutral ("they" instead of "he").

### CODE REVIEW

File: security.md [hunk 1]
```diff
@@ -189,15 +191,15 @@ Using git would allow you to add outsource/freelance developers for a limited ti
...
 
-* Some (small) companies define that all developers are admins. Your auditor might accept that and add a comment about that in his report.
+* Some (small) companies define that all developers are admins. Your auditor might accept that and add a comment about that in their report.
 
 ### Use change management for every production-affecting change
 
-* At this point you should already have automated testing, and (at least semi-) automatic of upgrading and downgrading production versions. The next step is to make sure the production system is immutable. Meaning, [a server that is once deployed, is never modified](https://martinfowler.com/bliki/ImmutableServer.html), merely replaced with a new updated instance. Any change of code, database, toggles must go through change management (like a pull request, or similar system).
+* At this point you should already have automated testing, and (at least semi-) automatic of upgrading and downgrading production versions. The next step is to make sure the production system is immutable. Meaning, any change of code, database, toggles must go through change management (like a pull request, or similar system).
```
Comment (rjmunro) [inline security.md]: Why remove the link?

Also "immutable" means can **never** be modified, not "can never be modified without going through change management". The script to create the replacement is what goes through change management.

Comment (Saraislet) [inline security.md]: I don't recall making this change, and I agree that it should be reverted. I'm not sure how that happened and how I missed it in skimming the file changes. Thanks for catching this.

**Comment by itaifrenkel** [comment]
@rjmunro Is the immutable images section clearer now ? (CC @Saraislet )

**Comment by rjmunro** [comment]
@itaifrenkel Looks good to me.

**Pull request merged**